src/arviz_stats/summary.py

```diff
@@ -215,3 +218,136 @@ def summary(
+    group: str
+        Select a group to compute the ROPE. Defaults to “posterior”.
+    coords : dict, optional
+        Coordinates defining a subset over the selected group.
+    sample_dims : str or sequence of hashable, optional
+        Defaults to ``rcParams["data.sample_dims"]``
+    ci_prob : float, optional
+        Probability for the credible interval. Defaults to ``rcParams["stats.ci_prob"]``.
+    ci_kind : {"hdi", "eti"}, optional
+        Type of credible interval. Defaults to ``rcParams["stats.ci_kind"]``.
+        If `kind` is stats_median or all_median, `ci_kind` is forced to "eti".
+    rope_dim : str, default "rope_dim"
+        Name for the dimension containing the ROPE values. Only used when `rope`
+        is a :class:`~xarray.Dataset`
+
+    Returns
+    -------
+    xarray.Dataset
+
+    See Also
+    --------
+    arviz.summary : Compute summary statistics and or diagnostics.
+    arviz.hdi : Compute highest density interval (HDI).
+    arviz.eti : Compute equal tail interval (ETI).
+
+    Examples
+    --------
+    Apply the same ROPE to a subset of variables:
+
+    .. ipython::
+
+        In [1]: from arviz_base import load_arviz_data
+           ...: from arviz_stats import ci_in_rope
+           ...: data = load_arviz_data("centered_eight")
+           ...: ci_in_rope(data, var_names=["mu", "tau"], rope=(-0.5, 0.5))
+
+    Apply different ROPEs to each variable:
+
+    .. ipython::
+
+        In [1]: ci_in_rope(data, rope={"mu": (-0.5, 0.5), "tau": (0.1, 0.2), "theta": (-0.1, 0.1)})
+
+    References
+    ----------
+    .. [1] Kruschke. Doing Bayesian Data Analysis, Second Edition: A Tutorial with R,
+        JAGS, and Stan. Academic Press, 2014. ISBN 978-0-12-405888-0.
+    """
+    sample_dims = validate_dims(sample_dims)
+    sample_dim = "sample" if len(sample_dims) > 1 else sample_dims[0]
+
+    dataset = extract(
+        data,
+        var_names=var_names,
+        filter_vars=filter_vars,
+        group=group,
+        sample_dims=sample_dims,
+        combined=True,
+        keep_dataset=True,
+    )
+
+    if isinstance(rope, dict):
+        if not all(var in dataset.data_vars for var in rope.keys()):
+            raise ValueError("`rope` must be a subset of the variables in `data`")
+        if not all(isinstance(v, tuple) and len(v) == 2 for v in rope.values()):
+            raise ValueError("`rope` must be a dict of tuples of length 2")
+    elif isinstance(rope, xr.Dataset):
+        if rope_dim not in rope.dims:
+            raise ValueError(f"{rope_dim} is not a dimension of `rope`")
+        if (rope_len := rope.sizes[rope_dim]) != 2:
+            raise ValueError(f"Length of {rope_dim} dim must be 2 but is {rope_len}")
+    else:
+        try:
+            rope = np.array(rope)
+        except ValueError as err:
+            raise ValueError(
+                "`rope` must be a dict, Dataset or array-like, failed to convert to array"
+            ) from err
```

Comment:
I have no problem restricting to tuple, but I really think dataset should also be a valid input

Reply:
ok